apps/test-suite/tests/TaskManager.js

```diff
@@ -105,6 +105,12 @@ export async function test(t) {
       });
     });
 
+    /*
```

Comment:
iirc this was non-deterministic & not necessarily related to this specific test (since an earlier one failed with the same error after we commented this out) -- am i remembering that correctly?

Reply:
Yeah that's true, I reran the tests and got the same error with another test. I've uncommented that test but left the description and moved requiring `TaskManager.js` to the `!isInCI` block in the latest commit.